test/test_integration.py

```diff
@@ -0,0 +1,114 @@
+# coding: utf-8
+
+#
+# Copyright 2014-2017 Groupon, Inc.
+# Copyright 2014-2017 The Billing Project, LLC
+#
+# The Billing Project, LLC licenses this file to you under the Apache License, version 2.0
+# (the "License"); you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at:
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+"""
+    Kill Bill
+
+    Kill Bill is an open-source billing and payments platform  # noqa: E501
+
+    OpenAPI spec version: 1.0.0
+    
+    Generated by: https://github.com/swagger-api/swagger-codegen.git
+"""
+
+
+from __future__ import absolute_import
+
+import unittest
+
+import killbill
+from killbill.models.account import Account  # noqa: E501
+from killbill.models.tenant import Tenant  # noqa: E501
+from killbill.models.payment_method import PaymentMethod  # noqa: E501
+from killbill.models.subscription import Subscription  # noqa: E501
+from killbill.models.invoice_dry_run import InvoiceDryRun  # noqa: E501
+from killbill.rest import ApiException
+
+
+
+class TestIntegration(unittest.TestCase):
+    """Account unit test stubs"""
+
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+    def test_integration(self):
+
+        # Create tenant
+        # TODO WORKING WITH ISSUES*! [ValueError: Invalid value for `api_key`, must not be `None`]
+        tenant = killbill.api.TenantApi()
+        body = Tenant(api_key='bob_5', api_secret='lazar_5')
+        tenant.create_tenant(body, 'test')
+
+        # Get tenant
+        # TODO NOT WORKING! [ApiException: (401) Reason: Unauthorized] or [ValueError: Invalid value for `api_secret`, must not be `None`]
+        body = Tenant(api_key='bob')
```

Comment:
Both key and secret should always be passed (use the values generated above).

Reply:
If I pass 2 arguments Python throws the following error: 
`TypeError: get_tenant_by_api_key() takes exactly 1 argument (2 given)` 
[Reff](https://github.com/wwjfrodriguez/killbill-client-python/blob/1915bb216d3a67b2309f68ba6120e386ddd9e02d/killbill/api/tenant_api.py#L1435)